### PR TITLE
remove amp-user-notification from experiment

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by erwinmombay** [comment]
meant remove from experiment* (not canary)

**Comment by cramforce** [comment]
LGTM

File: extensions/amp-user-notification/0.1/amp-user-notification.js [hunk 1]
```diff
@@ -82,14 +82,6 @@ class NotificationInterface {
  */
 export class AmpUserNotification extends AMP.BaseElement {
 
-  /**
-   * @return {boolean}
-   * @private
-   */
-  isExperimentOn_() {
-    return isExperimentOn(this.getWin(), EXPERIMENT);
```
Comment (dvoytenko) [inline extensions/amp-user-notification/0.1/amp-user-notification.js]: `EXPERIMENT` should be removed as well, right?

**Comment by dvoytenko** [comment]
LGTM with a small comment.

File: extensions/amp-user-notification/0.1/amp-user-notification.js (hunk 1 above)
Comment (erwinmombay) [inline extensions/amp-user-notification/0.1/amp-user-notification.js]: @dvoytenko yep! linter caught it too, thanks!

Comment (erwinmombay) [inline extensions/amp-user-notification/0.1/amp-user-notification.js]: done.

**Pull request merged**